### PR TITLE
Add ActionBar examples to TableView and ListView docs

### PR DESCRIPTION
**Pull request by reidbarber**
## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

File: packages/@react-spectrum/table/docs/TableView.mdx [hunk 1]
```diff
@@ -500,6 +504,70 @@ This behavior is slightly different in the highlight selection style, where sing
...
+    {id: 1, name: 'Charizard', type: 'Fire, Flying', level: '67'},
+    {id: 2, name: 'Blastoise', type: 'Water', level: '56'},
+    {id: 3, name: 'Venusaur', type: 'Grass, Poison', level: '83'},
+    {id: 4, name: 'Pikachu', type: 'Electric', level: '100'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([1]));
+
+  return (
+    <ActionBarContainer>
```
Comment (snowystinger) [inline packages/@react-spectrum/table/docs/TableView.mdx:531]: lets add a height like the ListView example has

**Review by snowystinger — DISMISSED** [review]

File: packages/@react-spectrum/list/docs/ListView.mdx [hunk 2]
```diff
@@ -363,6 +366,55 @@ This behavior is slightly different in the highlight selection style, where sing
 </Flex>
 ```
 
+
+### Action bar <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+An ActionBar can be used when a user needs to perform actions on one or more items at the same time.
```
Comment (ktabors) [inline packages/@react-spectrum/list/docs/ListView.mdx:372]: Should ActionBar be a link to its docs?

Comment (reidbarber) [reply packages/@react-spectrum/list/docs/ListView.mdx:372]: We should definitely add that after the ActionBar docs get merged and are public. These might go out before that though.

File: packages/@react-spectrum/list/docs/ListView.mdx [hunk 3]
```diff
@@ -363,6 +366,55 @@ This behavior is slightly different in the highlight selection style, where sing
...
+  let items = [
+    {key: 0, name: 'Adobe Photoshop'},
+    {key: 1, name: 'Adobe Illustrator'},
+    {key: 2, name: 'Adobe XD'}
+  ];
+
+  let [selectedKeys, setSelectedKeys] = React.useState(new Set([0]));
+
+  return (
+    <ActionBarContainer height={300} maxWidth="size-6000">
```
[the remark this answers is not in the document]
Comment (reidbarber) [reply packages/@react-spectrum/list/docs/ListView.mdx:387]: Updated to remove the ListView's maxWidth

**Review by snowystinger — APPROVED** [review]

**Review by ktabors — APPROVED** [review]

**Comment by reidbarber** [comment]
I've added a link to the ActionBar docs page, so this should get merged after https://github.com/adobe/react-spectrum/pull/2777.

**Review by LFDanLu — APPROVED** [review]
LGTM

**Review by ktabors — APPROVED** [review]

File: packages/@react-spectrum/list/docs/ListView.mdx (hunk 2 above)
Comment (reidbarber) [reply packages/@react-spectrum/list/docs/ListView.mdx]: Added back since we decided we'll merge this after the ActionBar docs are public

**Comment by reidbarber** [comment]
Will merge after https://github.com/adobe/react-spectrum/pull/2777

**Pull request merged**